project_generator/commands/generate.py

```diff
@@ -27,6 +29,10 @@ def run(args):
     generated = True
     for project in generator.generate(args.project):
         generated = False
+        if project.workspace_name is not None:
+            logger.info("Generating %s for %s in workspace %s", args.tool, project.name, project.workspace_name)
```

Comment:
Travis failes `AttributeError: ProjectWorkspace instance has no attribute 'workspace_name'`

Reply:
Whoopsie! I didn't even check the Travis results…